{
  "Faults": [
    {
      "latitude": 51.35338864463416,
      "type": "LV",
      "reference": "KV3849",
      "etaUpdate": true,
      "engineerOnSiteTime": "2023-04-30T07:15:00.000+0000",
      "affectedCustomerCount": 9,
      "networkId": "com.sse.ssepd.sepd",
      "UUID": "fa566eee-509e-45c3-8070-2de1b21a4473",
      "minLongitude": -1.9976,
      "py/class": "http://opengrid.com/givr/1.0#//Fault",
      "longitude": -1.9956429579303772,
      "resolved": false,
      "py/object": "Fault",
      "srtUpdate": true,
      "loggedAt": "2023-04-30T06:12:00.000+0000",
      "estimatedRestoration": "2023-04-30T21:00:00.000+0000",
      "message": "We’re sorry for the loss of supply. We currently have a fault affecting the areas listed. Our engineers are on site working hard to get the power back on as quickly as they can. If you need more information, please call us on 105 or send us a message on <a href=\"http://m.me/ssencommunity/\">Facebook</a> or Twitter (<a href=\"https://twitter.com/ssencommunity\">@ssencommunity</a>) and quote reference 'KV3849'",
      "uri": "http://api.sse.com/powerdistribution/network/v3/api/fault/KV3849",
      "maxLongitude": -1.9936,
      "minLatitude": 51.3521,
      "name": "SN10 Area",
      "otherUpdate": true,
      "affectedAreas": [
        "SN10",
        "SN10 1DU",
        "SN10 1EB",
        "SN10 1EX",
        "SN10 1FG",
        "SN10 1HT",
        "SN10 1HU",
        "SN10 1JG",
        "SN10 1JQ"
      ],
      "location": {
        "coordinates": [
          [
            [
              -1.9976,
              51.3558
            ],
            [
              -1.9936,
              51.3558
            ],
            [
              -1.9936,
              51.3521
            ],
            [
              -1.9976,
              51.3521
            ],
            [
              -1.9976,
              51.3558
            ]
          ]
        ],
        "type": "Polygon"
      },
      "estimatedArrivalOnSiteTime": "2023-04-30T07:00:00.000+0000",
      "region": "com.opengrid.geo.world.europe.gb",
      "maxLatitude": 51.3558,
      "etrUpdate": true
    },
    {
      "latitude": 50.74505099719263,
      "type": "LV",
      "reference": "KV4010",
      "etaUpdate": true,
      "engineerOnSiteTime": "2023-04-30T14:25:00.000+0000",
      "affectedCustomerCount": 37,
      "networkId": "com.sse.ssepd.sepd",
      "UUID": "6f534271-6925-4a2d-a896-c2106c1a4731",
      "minLongitude": -1.6254,
      "py/class": "http://opengrid.com/givr/1.0#//Fault",
      "longitude": -1.5954290643737759,
      "resolved": false,
      "py/object": "Fault",
      "srtUpdate": true,
      "loggedAt": "2023-04-30T13:34:00.000+0000",
      "estimatedRestoration": "2023-04-30T22:59:00.000+0000",
      "message": "We’re sorry for the loss of supply. We currently have a fault affecting the areas listed. Our engineers are on site working hard to get the power back on as quickly as they can. If you need more information, please call us on 105 or send us a message on <a href=\"http://m.me/ssencommunity/\">Facebook</a> or Twitter (<a href=\"https://twitter.com/ssencommunity\">@ssencommunity</a>) and quote reference 'KV4010'",
      "uri": "http://api.sse.com/powerdistribution/network/v3/api/fault/KV4010",
      "maxLongitude": -1.5907,
      "minLatitude": 50.7381,
      "name": "SO41 Area",
      "otherUpdate": true,
      "affectedAreas": [
        "SO41 0DB",
        "SO41 0EH",
        "SO41 0JX",
        "SO41 0JZ",
        "SO41 0LA",
        "SO41 0LD",
        "SO41 0LL",
        "SO41 0LN",
        "SO41 0LW",
        "SO41 0UA",
        "SO41 0ZA",
        "SO41 0ZB",
        "SO41 0ZD",
        "SO41 0ZF",
        "SO41 0ZG",
        "SO41 0ZH",
        "SO41 0ZJ"
      ],
      "location": {
        "coordinates": [
          [
            [
              -1.6254,
              50.7516
            ],
            [
              -1.5907,
              50.7516
            ],
            [
              -1.5907,
              50.7381
            ],
            [
              -1.6254,
              50.7381
            ],
            [
              -1.6254,
              50.7516
            ]
          ]
        ],
        "type": "Polygon"
      },
      "estimatedArrivalOnSiteTime": "2023-04-30T14:10:00.000+0000",
      "region": "com.opengrid.geo.world.europe.gb",
      "maxLatitude": 50.7516,
      "etrUpdate": true
    },
    {
      "latitude": 50.84336300084347,
      "type": "LV",
      "reference": "KV3999",
      "etaUpdate": true,
      "engineerOnSiteTime": "2023-04-30T13:40:00.000+0000",
      "affectedCustomerCount": 17,
      "networkId": "com.sse.ssepd.sepd",
      "UUID": "d90b0d0d-5949-4dc6-8350-d850997e24b8",
      "minLongitude": -1.803,
      "py/class": "http://opengrid.com/givr/1.0#//Fault",
      "longitude": -1.7929539635243679,
      "resolved": false,
      "py/object": "Fault",
      "srtUpdate": true,
      "loggedAt": "2023-04-30T13:00:00.000+0000",
      "estimatedRestoration": "2023-04-30T22:00:00.000+0000",
      "message": "We’re sorry for the loss of supply. We currently have a fault affecting the areas listed. Our engineers are on site working hard to get the power back on as quickly as they can. If you need more information, please call us on 105 or send us a message on <a href=\"http://m.me/ssencommunity/\">Facebook</a> or Twitter (<a href=\"https://twitter.com/ssencommunity\">@ssencommunity</a>) and quote reference 'KV3999'",
      "uri": "http://api.sse.com/powerdistribution/network/v3/api/fault/KV3999",
      "maxLongitude": -1.7878,
      "minLatitude": 50.8298,
      "name": "BH24 Area",
      "otherUpdate": true,
      "affectedAreas": [
        "BH24 1EJ",
        "BH24 1EQ"
      ],
      "location": {
        "coordinates": [
          [
            [
              -1.803,
              50.8451
            ],
            [
              -1.7878,
              50.8451
            ],
            [
              -1.7878,
              50.8298
            ],
            [
              -1.803,
              50.8298
            ],
            [
              -1.803,
              50.8451
            ]
          ]
        ],
        "type": "Polygon"
      },
      "region": "com.opengrid.geo.world.europe.gb",
      "maxLatitude": 50.8451,
      "etrUpdate": true
    },
    {
      "latitude": 51.370893882693295,
      "type": "HV",
      "reference": "KV4074",
      "etaUpdate": true,
      "affectedCustomerCount": 9,
      "networkId": "com.sse.ssepd.sepd",
      "UUID": "ebb08cee-d061-4ae9-a4ae-fc210fb1288a",
      "minLongitude": -2.1425,
      "py/class": "http://opengrid.com/givr/1.0#//Fault",
      "longitude": -2.139024529323626,
      "resolved": false,
      "py/object": "Fault",
      "srtUpdate": true,
      "loggedAt": "2023-04-30T16:12:00.000+0000",
      "estimatedRestoration": "2023-05-01T01:00:00.000+0000",
      "message": "We’re sorry for the loss of supply. We currently have a fault affecting the areas listed. Our engineers are on site working hard to get the power back on as quickly as they can. If you need more information, please call us on 105 or send us a message on <a href=\"http://m.me/ssencommunity/\">Facebook</a> or Twitter (<a href=\"https://twitter.com/ssencommunity\">@ssencommunity</a>) and quote reference 'KV4074'",
      "uri": "http://api.sse.com/powerdistribution/network/v3/api/fault/KV4074",
      "maxLongitude": -2.1379,
      "minLatitude": 51.3704,
      "name": "SN12 Area",
      "otherUpdate": true,
      "affectedAreas": [
        "SN12 6ES"
      ],
      "location": {
        "coordinates": [
          [
            [
              -2.1425,
              51.3725
            ],
            [
              -2.1379,
              51.3725
            ],
            [
              -2.1379,
              51.3704
            ],
            [
              -2.1425,
              51.3704
            ],
            [
              -2.1425,
              51.3725
            ]
          ]
        ],
        "type": "Polygon"
      },
      "region": "com.opengrid.geo.world.europe.gb",
      "maxLatitude": 51.3725,
      "etrUpdate": true
    },
    {
      "latitude": 50.744138343164195,
      "type": "HV",
      "reference": "KV4145",
      "etaUpdate": true,
      "affectedCustomerCount": 90,
      "networkId": "com.sse.ssepd.sepd",
      "UUID": "32235c26-8ec3-434f-9856-e377683a3eac",
      "minLongitude": -1.5886,
      "py/class": "http://opengrid.com/givr/1.0#//Fault",
      "longitude": -1.54771628804375,
      "resolved": false,
      "py/object": "Fault",
      "srtUpdate": true,
      "loggedAt": "2023-04-30T17:16:00.000+0000",
      "estimatedRestoration": "2023-04-30T23:30:00.000+0000",
      "message": "We’re sorry for the loss of supply. We currently have a fault affecting the areas listed. Our engineers are on site working hard to get the power back on as quickly as they can. If you need more information, please call us on 105 or send us a message on <a href=\"http://m.me/ssencommunity/\">Facebook</a> or Twitter (<a href=\"https://twitter.com/ssencommunity\">@ssencommunity</a>) and quote reference 'KV4145'",
      "uri": "http://api.sse.com/powerdistribution/network/v3/api/fault/KV4145",
      "maxLongitude": -1.518,
      "minLatitude": 50.7225,
      "name": "SO41 Area",
      "otherUpdate": true,
      "affectedAreas": [
        "SO41 0JL",
        "SO41 8AA",
        "SO41 8AB",
        "SO41 8AD",
        "SO41 8AE",
        "SO41 8AF",
        "SO41 8AG",
        "SO41 8AH",
        "SO41 8AJ",
        "SO41 8AL",
        "SO41 8AN",
        "SO41 8FF",
        "SO41 8FJ",
        "SO41 8FL",
        "SO41 8FT",
        "SO41 8QZ"
      ],
      "location": {
        "coordinates": [
          [
            [
              -1.5886,
              50.7512
            ],
            [
              -1.518,
              50.7512
            ],
            [
              -1.518,
              50.7225
            ],
            [
              -1.5886,
              50.7225
            ],
            [
              -1.5886,
              50.7512
            ]
          ]
        ],
        "type": "Polygon"
      },
      "region": "com.opengrid.geo.world.europe.gb",
      "maxLatitude": 50.7512,
      "etrUpdate": true
    },
    {
      "latitude": 51.54854893415319,
      "type": "LV",
      "reference": "KV3964",
      "etaUpdate": true,
      "engineerOnSiteTime": "2023-04-30T13:00:00.000+0000",
      "affectedCustomerCount": 22,
      "networkId": "com.sse.ssepd.sepd",
      "UUID": "7b33afd2-a4b3-42b0-a6be-bfd7f34b157a",
      "minLongitude": -0.3575,
      "py/class": "http://opengrid.com/givr/1.0#//Fault",
      "longitude": -0.35449614153921777,
      "resolved": false,
      "py/object": "Fault",
      "srtUpdate": true,
      "loggedAt": "2023-04-30T11:28:00.000+0000",
      "estimatedRestoration": "2023-04-30T21:30:00.000+0000",
      "message": "We’re sorry for the loss of supply. We currently have a fault affecting the areas listed. Our engineers are on site working hard to get the power back on as quickly as they can. If you need more information, please call us on 105 or send us a message on <a href=\"http://m.me/ssencommunity/\">Facebook</a> or Twitter (<a href=\"https://twitter.com/ssencommunity\">@ssencommunity</a>) and quote reference 'KV3964'",
      "uri": "http://api.sse.com/powerdistribution/network/v3/api/fault/KV3964",
      "maxLongitude": -0.3518,
      "minLatitude": 51.5472,
      "name": "UB5 Area",
      "otherUpdate": true,
      "affectedAreas": [
        "UB5 4RB",
        "UB5 4RD",
        "UB5 4SG",
        "UB5 4SU"
      ],
      "location": {
        "coordinates": [
          [
            [
              -0.3575,
              51.5496
            ],
            [
              -0.3518,
              51.5496
            ],
            [
              -0.3518,
              51.5472
            ],
            [
              -0.3575,
              51.5472
            ],
            [
              -0.3575,
              51.5496
            ]
          ]
        ],
        "type": "Polygon"
      },
      "region": "com.opengrid.geo.world.europe.gb",
      "maxLatitude": 51.5496,
      "etrUpdate": true
    }
  ]
}
